client-app/app/templates/index.hbs

```diff
@@ -114,6 +123,24 @@
           </LinkTo>
         {{/if}}
 
+        {{#if this.buildingGroupingRegex}}
+          <button
+            class="settings btn"
+            type="button"
+            {{on "click" this.createGroupingRegexFromSelectedRows}}
+          >
+            <span>Create Grouping Pattern</span>
+          </button>
+        {{else}}
+          <button
+            class="settings btn"
+            type="button"
+            {{on "click" this.buildGroupingRegexFromSelectedRows}}
+          >
+            <span>Build Grouping Pattern</span>
+          </button>
+        {{/if}}
```

Comment:
The grouping feature isn't always enabled. e.g. on our hosting it's disabled for multisite tenants. So we need to hide the button in that case. Maybe checking `this.showSettings` is enough?